### PR TITLE
Increase coverage

### PR DESCRIPTION
**Pull request by cpehle**

### CODE REVIEW

File: norse/torch/functional/test/test_threshhold.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        out = sign(x, method, alpha)
+        assert torch.equal(out, torch.ones(10))
+
+        x = torch.full((10,), -0.1)
+        out = sign(x, method, alpha)
+        assert torch.equal(out, -torch.ones(10))
+
+
+if __name__ == "__main__":
+    test_threshhold()
```
Comment (Jegp) [inline norse/torch/functional/test/test_threshhold.py:57]: Spelling error

File: norse/torch/functional/test/test_threshhold.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        x = torch.full((10,), 0.1)
+        out = sign(x, method, alpha)
+        assert torch.equal(out, torch.ones(10))
+
+        x = torch.full((10,), -0.1)
+        out = sign(x, method, alpha)
+        assert torch.equal(out, -torch.ones(10))
+
+
+if __name__ == "__main__":
```
Comment (Jegp) [inline norse/torch/functional/test/test_threshhold.py:56]: Why are you adding this? Pytest should catch every `test_*` function recursively

**Review by Jegp — CHANGES_REQUESTED** [review]
Nice work! There's a small bug in the test_treshold.py file.

File: norse/torch/functional/test/test_threshhold.py (hunk 1 above)
Comment (cpehle) [reply norse/torch/functional/test/test_threshhold.py]: Good catch

File: norse/torch/functional/test/test_threshhold.py (hunk 2 above)
Comment (cpehle) [reply norse/torch/functional/test/test_threshhold.py]: Removed

**Review by Jegp — APPROVED** [review]

**Comment by Jegp** [comment]
I'm happy as a tornado in a trailer park! :tornado: LGTM

**Pull request merged**